.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Retrieve screenshots
         if: failure()
         run: .circleci/process-artifacts.sh
-        timeout-minutes: 3
+        timeout-minutes: 1
```

Comment:
why reduce time?

Reply:
Just matching another repo. This step should never take more than a few seconds.